document_page_approval/models/document_page_history.py

```diff
@@ -151,8 +151,9 @@ def _compute_page_url(self):
             )
 
             page.page_url = (
-                "{}/web#db={}&id={}&" "model=document.page.history"
-            ).format(base_url, self.env.cr.dbname, page.id)
+                f"{base_url}/web#db={self.env.cr.dbname}&id={page.id}&"
+                "model=document.page.history"
```

Comment:
Nota that these URLs are still supported, but have changed and might stop working in future versions.